packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -585,15 +585,13 @@ end
 
 local function periodically_refresh_cache(auth_token)
     -- This function is invoked from within init_worker_by_lua code.
-    -- ngx.timer.at() can be called here, whereas most of the other ngx.*
-    -- API is not available.
+    -- ngx.timer.every() is called here, a more robust alternative to
+    -- ngx.timer.at() as suggested by the openresty/lua-nginx-module
+    -- documentation: 
```

Comment:
Can we also link here to Jira issue that made us switch from `.at` to `.every`, please?

Reply:
ack, done